interface/wx/intl.h

```diff
@@ -390,7 +390,10 @@ class wxLocale
     bool AddCatalog(const wxString& domain, wxLanguage msgIdLanguage);
 
     /**
-        Calls wxTranslations::AddCatalog(const wxString&, wxLanguage, const wxString&).
+        Calls wxTranslations::AddCatalog(const wxString&, const wxString&).
+
+        This overload shouldn't be used any longer as @a msgIdCharset is
+        just ignored.
```

Comment:
Shouldn't it also be deprecated, then, to avoid confusion?

Reply:
I've hesitated to do it, as it does no harm to keep it, but done now, hopefully it's not used much anyhow.